src/models/Expense.model.js

```diff
@@ -1,9 +1,46 @@
 'use strict';
 
+const { DataTypes, Sequelize } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const Expense = sequelize.define(
-  // your code goes here
+  'Expense',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
```

Comment:
Ensure that the `userId` field is set up as a foreign key if it references the `Users` table to maintain referential integrity.